extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -216,7 +216,7 @@ export class AmpAnalytics extends AMP.BaseElement {
 
     if (!this.config_['triggers']) {
       const TAG = this.getName_();
-      user().error(TAG, 'No triggers were found in the ' +
+      user(this.element).error(TAG, 'No triggers were found in the ' +
```

Comment:
to make the API simpler, we can take `BaseElement` instead of `Element`. So user can do
```
user(this).error(...
```